library/modules/bigip_device_auth.py

```diff
@@ -119,6 +119,19 @@
     choices:
```

Comment:
Noticed port sub-option on line 55 says default: 49163 - is this still true?

Reply:
Yeah I need to fix that!

Reply:
done